app/controllers/course/forum/topics_controller.rb

```diff
@@ -73,4 +74,10 @@ def topic_params
   def load_topic
     @topic ||= @forum.topics.friendly.find(params[:id])
   end
+
+  def add_forum_topic_item_breadcrumb
+    add_breadcrumb @forum.name, course_forum_path(current_course, @forum)
```

Comment:
I've answered this question before. Can not put this in parent controller, because you can not load `@forum` in every situations. For example, you need to manually load forum using slug when the forum exists, meanwhile, you need to use `load_resource` when you create or new a forum in `ForumController`